### PR TITLE
chore: fixed a few cal.ai ui issues and cal.video logo

### LINKED ISSUE #15042: [CAL-3707] cal video logo placement
null

<sub>[CAL-3707](https://linear.app/calcom/issue/CAL-3707/cal-video-logo-placement)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
- moved logo back to the left (not center. because when recording, the recording time is in the center overlapping)
- removed toast
- toggling cal.ai logo instead (making it red)

### CODE REVIEW

**Review by Udit-takkar — DISMISSED** [review]
LGTM. Tested and worked fine

File: apps/web/modules/videos/ai/ai-transcribe.tsx [hunk 1]
```diff
@@ -26,11 +25,25 @@ export const CalAiTranscribe = () => {
   );
 
   useDailyEvent("transcription-started", (ev) => {
-    showToast(t("transcription_enabled"), "success");
+    daily?.updateCustomTrayButtons({
+      transcription: {
+        label: "Stop",
+        tooltip: "Toggle real time transcription powered by AI",
```
Comment (PeerRich) [inline apps/web/modules/videos/ai/ai-transcribe.tsx:31]:
```suggestion
        tooltip: "Transcription powered by AI",
```

CleanShot 2024-05-17 at 11 02 57@2x

File: apps/web/modules/videos/ai/ai-transcribe.tsx [hunk 2]
```diff
@@ -26,11 +25,25 @@ export const CalAiTranscribe = () => {
...
+      },
+    });
   });
 
   useDailyEvent("transcription-stopped", (ev) => {
-    showToast(t("transcription_stopped"), "success");
+    daily?.updateCustomTrayButtons({
+      transcription: {
+        label: "Cal.ai",
+        tooltip: "Toggle real time transcription powered by AI",
```
Comment (PeerRich) [inline apps/web/modules/videos/ai/ai-transcribe.tsx:42]:
```suggestion
        tooltip: "Transcription powered by AI",
```

File: packages/features/bookings/Booker/Booker.tsx [hunk 3]
```diff
@@ -327,7 +327,7 @@ const BookerComponent = ({
                 {!hideEventTypeDetails && orgBannerUrl && !isPlatform && (
                   <img
                     loading="eager"
-                    className="-mb-9 h-28 max-h-28 rounded-tl-md sm:max-h-24"
```
Comment (Udit-takkar) [inline packages/features/bookings/Booker/Booker.tsx:330]: Does this fix the org banner responsive bug?

Comment (Udit-takkar) [reply packages/features/bookings/Booker/Booker.tsx:330]: https://github.com/calcom/cal.com/issues/14937

Comment (Udit-takkar) [reply packages/features/bookings/Booker/Booker.tsx:330]: Not sure if we should remove max height as user can upload banner with bigger height (even though we mention the recommended format is 1500x500)

**Review by Udit-takkar — APPROVED** [review]
LGTM

**Pull request merged**